channeldb/channel.go

```diff
@@ -738,6 +757,11 @@ type OpenChannel struct {
 	// have private key isolation from lnd.
 	RevocationKeyLocator keychain.KeyLocator
 
+	// OtherShortChannelID is another ShortChannelID that this channel may
+	// be referred to. This is only populated for option_scid_alias
+	// channels and will be the empty ShortChannelID otherwise.
+	OtherShortChannelID lnwire.ShortChannelID
```

Comment:
The switch has two indices, `alias->real` and `real->alias`. I named it Other since if this is an option_scid_alias channel w/o zero-conf, then this is the alias. And in the zero-conf case, it's the real scid. It's a bit confusing